e2e/command_test.go

```diff
@@ -82,7 +92,99 @@ func TestE2E_MetaCommandErrors(t *testing.T) {
 		}
 	}
 
-	delete()
+	deleteTemp()
+}
+
+// TestE2E_TLSErrors tests the CLI to disable an enabled task. This
+// starts up a local Consul server and runs CTS with TLS in dev mode.
+func TestE2E_TLSErrors(t *testing.T) {
+	t.Parallel()
+
+	srv := newTestConsulServer(t)
+	defer srv.Stop()
+
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "command_tls_errors")
+
+	tlsc := tlsConfig{
```

Comment:
You could make a defaultTLSConfig method that returns this value. I'd also suggest having no caCert set for the default CTS TLS config, and when we add mutualTLS, we could add defaultMTLSConfig that has caCert.